core/src/main/java/org/mskcc/cbio/portal/dao/DaoReferenceGenome.java

```diff
@@ -85,13 +96,13 @@ public static void addReferenceGenome(ReferenceGenome referenceGenome, boolean o
      * Deletes all Reference Genomes.
      * @throws DaoException Database Error.
      */
-    public static void deleteAllRecords() throws DaoException {
+    public void deleteAllRecords() throws DaoException {
 
         Connection con = null;
         PreparedStatement pstmt = null;
         ResultSet rs = null;
         try {
-            con = JdbcUtil.getDbConnection(DaoCancerStudy.class);
+            con = JdbcUtil.getDbConnection(DaoReferenceGenome.class);
```

Comment:
a TRUNCATE won't normally work since there are foreign keys to this table, right?